app/assets/src/components/views/SampleViewV2/ReportTable.jsx

```diff
@@ -333,11 +333,13 @@ class ReportTable extends React.Component {
 
   renderNtNrDecimalValues = ({ cellData, decimalPlaces }) => {
     return this.renderNtNrStack({
-      cellData: cellData.map(val =>
-        TableRenderers.formatNumberWithCommas(
-          Number(val).toFixed(decimalPlaces || 0)
-        )
-      ),
+      cellData: cellData.map(val => {
+        let needsTruncation = val % 1 !== 0; // Checks if a number has a decimal place/is a whole number
```

Comment:
@ovalenzuela19 this function simply ignores a decimalPlaces input argument which is not good. It is also a generic function that can be used by multiple columns with different requirements. 
The effect of rounding is not unintended. 
We need further discussion on the solution.

Reply:
Thanks @tfrcarvalho! I can’t believe I missed that argument. I’ll look for alternative solutions in the morning.